lib/util/length-based-splitter.js

```diff
@@ -0,0 +1,76 @@
+import Stream from 'stream';
+
+class LengthBasedSplitter extends Stream.Transform {
```

Comment:
Why is this in `util` and not `transformer`?

Reply:
I'm gonna move it to util/transformer as it will be reused in the future for different Stream implementations